src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -597,7 +632,10 @@ protected virtual string GenerateDisplayText(ModelMetadata metadata)
             return tagBuilder.ToHtmlString(TagRenderMode.Normal);
         }
 
-        protected virtual HtmlString GenerateEditor(ModelMetadata metadata, string htmlFieldName, string templateName,
+        protected virtual HtmlString GenerateEditor(
+            ModelMetadata metadata,
+            string htmlFieldName,
```

Comment:
Shouldn't this be an expression since it eventually gets passed into `GetFullHtmlFieldName`?